packages/lab/src/tabs-next/TabstripNext.css

```diff
@@ -0,0 +1,138 @@
+/* Component class applied to the root element */
+.saltTabstripNext {
+  --tabs-tabstrip-height: var(--saltTabs-tabstrip-height, var(--salt-size-stackable));
+  --tabs-tabstrip-dragging-display: none;
+  --tabs-tabstrip-display: inline-flex;
+  --tabs-tabstrip-background: transparent;
+
+  --saltTabs-activationIndicator-height: 1px;
+  --tabs-activationIndicator-background: var(--salt-container-primary-borderColor);
+  --tabs-activationIndicator-thumb-height: 2px;
```

Comment:
```suggestion
  --tabs-activationIndicator-thumb-height: var(--salt-editable-borderWidth-active);
```

Reply:
not sure if this makes sense but i assume it is and the other 1px above can be salt-size-border